src/lib/components/utils/acm-button.svelte

```diff
@@ -3,29 +3,24 @@
   export let text = 'Submit';
 </script>
 
-<a
-  class="call-to-action headers"
-  role="button"
-  href={link}
-  target="_blank"
-  rel="noopener norefferer">{text}</a
->
+<a class="brand-header size-md" role="button" href={link} target="_blank" rel="noopener norefferer">
+  {text}
+</a>
 
 <style lang="scss">
-  .call-to-action {
+  a {
     display: block;
     text-align: center;
-    cursor: pointer;
-    user-select: none;
-    text-decoration: none;
-    border-radius: 12px;
     padding: 12px 24px;
-    margin-top: 64px;
-    background-color: var(--acm-dark);
-    color: var(--acm-light);
-    transition: color 0.25s ease-in-out;
-  }
-  .call-to-action:hover {
-    color: var(--acm-blue);
+    background-color: var(--button-bg);
+    color: var(--perma-light);
+    border-radius: 12px;
+    text-decoration: none;
+    user-select: none;
+    transition: 0.25s ease-in-out;
+
+    &:hover {
+      background-color: var(--button-hover);
```

Comment:
button-hover color changes color depending on theme. In light mode, button-hover color will be acmBlue. In dark mode, it'll be a light gray. This is to help with contrast between themes.